### PR TITLE
docs: SLA

### PR DESCRIPTION
**Pull request by hrwX**

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/support/issue.md [hunk 1]
```diff
@@ -16,6 +16,18 @@ Frappe system.
 
 <img class="screenshot" alt="Issue" src="{{docs_base_url}}/assets/img/support/issue.png">
 
+#### Service Level Agreement
+
+If Service Level Agreements are enabled, you can track it in Issues. Every Issue
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/support/issue.md]: The GIFs have inconsistent padding, crop them to be the same as screenshots i.e., just a little on either side
Use realistic data for the gifs and screenshots, use realistic names, issues, data etc

File: erpnext_com/www/docs/user/manual/en/support/service-level-agreement.md [hunk 2]
```diff
@@ -16,35 +16,37 @@ Steps for creating a Employee Group:
...
+ * Priorities : Contains multiple Issue Priorities and their Time to respond and resolve (eg: (3, days) or (3, hours)) that are defined in Service Level.
+ * Support days : Contains the Days of the week on which Support is provided and has a Start Time and End Time of the Working day.
 
 ## Service Level Agreement
 
 A service level agreement (SLA) is a contract between a service provider (either internal or external) and the end user that defines the level of service expected from the service provider. SLAs are output-based in that their purpose is specifically to define what the customer will receive. SLAs do not define how the service itself is provided or delivered.
 
 To create a new Service Level Agreement go to:
-
+> Enable Track Service Level Agreement in Support Settings
```
[the remark this answers is not in the document]
Comment (hrwX) [reply erpnext_com/www/docs/user/manual/en/support/service-level-agreement.md]: "Track Service Level Agreement" is the check in Support Settings

File: erpnext_com/www/docs/user/manual/en/support/issue.md [hunk 3]
```diff
@@ -16,13 +16,25 @@ Frappe system.
 
 <img class="screenshot" alt="Issue" src="{{docs_base_url}}/assets/img/support/issue.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/support/issue.md]: the GIFs have extra width, crop

Comment (hrwX) [reply erpnext_com/www/docs/user/manual/en/support/issue.md]: clubed both gifs into one, for better understanding

**Pull request merged**